graphs/find_connected_components.cpp

```diff
@@ -0,0 +1,65 @@
+#include "../headers.h"
+
+/* Get connected components in a graph */
+
+void get_connected_components(vector<vi>& g, int n)
+{
+	/* Creating the graph */
+	map<int, set<int>> graph;
```

Comment:
set is internally a BST as you know, it will be better to use vector in this case, as you want to iterate the graph. ( line number 35 will execute faster.

Reply:
I see your point! 👍🏻 and will take it. From a thought process, for the moment, I will stick with it